tests/python/frontend/tflite/test_forward.py

```diff
@@ -933,9 +933,24 @@ def representative_data_gen():
         is_float_output=True,
         int_quant_dtype=int_quant_dtype,
     )
+    # TFLite.Model.Model has changed to TFLite.Model from 1.14 to 2.1
+    try:
```

Comment:
Can you add a TODO to remove this when the relevant PR is merged (put a link to it)

Reply:
I will focus on it. Remove it later when relevant PR is merged by myself.